linter/.pylintrc

```diff
@@ -1,12 +1,21 @@
 [MASTER]
-load-plugins=linter.conanv2_transition
+load-plugins=linter.conanv2_transition,
+             linter.conanfile_transform
 py-version=3.6
 recursive=no
 suggestion-mode=yes
 unsafe-load-any-extension=no
 
 [MESSAGES CONTROL]
-disable=all
+disable=fixme,
```

Comment:
My plan is to disable _everything_ needed for all the recipes to work... then, following PRs can try to enable one by one.